libcodechecker/session_manager.py

```diff
@@ -8,16 +8,17 @@
 with a particular CodeChecker server.
 """
 
-from datetime import datetime
 import getpass
 import hashlib
 import json
 import os
-import portalocker
-import stat
 import shutil
+import stat
 import time
 import uuid
+from datetime import datetime
```

Comment:
This should stay at the front line. System imports are ordered by their package name.